mwdb/app.py

```diff
@@ -259,6 +261,12 @@ def require_auth():
 api.add_resource(FileDownloadResource, "/file/<hash64:identifier>/download")
 api.add_resource(FileDownloadZipResource, "/file/<hash64:identifier>/download/zip")
 
+# RelatedFiles endpoints
+api.add_resource(RelatedFileItemResource, "/related_file/<hash64:identifier>")
+api.add_resource(
+    RelatedFileDownloadResource, "/related_file/<hash64:identifier>/download"
+)
```

Comment:
I would put it in `/object` hierarchy so the access permission check would be also simplified.

```suggestion
api.add_resource(RelatedFileItemResource, "/<any(file, config, blob, object):type>/<hash64:identifier>/related_file/<hash64:related_file_identifier>")
api.add_resource(
    RelatedFileDownloadResource, "/<any(file, config, blob, object):type>/<hash64:identifier>/related_file/<hash64:related_file_identifier>/download"
)
```

Reply:
Done